src/react/hooks/use-presence-listener.ts

```diff
@@ -201,7 +201,13 @@ export const usePresenceListener = (params?: UsePresenceListenerParams): UsePres
         clearErrorState();
       })
       .catch((error: unknown) => {
-        logger.error('usePresenceListener(); error fetching initial presence data', { error, roomId: room.roomId });
+        const errorInfo = error as Ably.ErrorInfo;
+        if (errorInfoIs(errorInfo, ErrorCodes.RoomIsReleased)) return;
+
+        logger.error('usePresenceListener(); error fetching initial presence data', {
+          error,
+          roomId: room.roomId,
+        });
         setErrorState(error as Ably.ErrorInfo);
```

Comment:
nitpick: use the `errorInfo` var here to avoid casting again?

Reply:
Done!